demo/index.html

```diff
@@ -32,21 +32,21 @@
 
   <paper-toolbar>
     <paper-icon-button icon="menu"></paper-icon-button>
-    <span title>Toolbar</span>
+    <span toolbar-title>Toolbar</span>
```

Comment:
Should this be a class instead of an attribute?

Rationale:
 - custom attributes are not valid HTML for native elements unless prefixed with data-
 - the name collision that caused this bug would be avoided (`title` class is :ok: )
 - future attribute name collisions are still possible - what if the element I want to use as the `toolbar-title` has a `toolbarTitle` property of its own?